addons/mail/controllers/discuss/channel.py

```diff
@@ -47,6 +57,7 @@ def _process_request_for_all(self, store: Store, name, params):
             request.update_context(
                 channels=request.env.context["channels"] | channels, add_channels_last_message=True
             )
+            self._add_has_unpinned_channels_to_store(store=store)
```

Comment:
We usually always give store as positional.

Reply:
Changed, but I'd expect passing named param is more reliable in case of code change, i.e. minimal diff in case the signature of function has to be changed.